ansible/tasks/create_minio_bearer_token.yml

```diff
@@ -22,3 +22,5 @@
       data:
         bearer-token: "{{ minio_prom_bearer_token }}"
     body_format: json
+  vars:
```

Comment:
VENV containing vault module not needed to execute this ansible command.
We should remove this var from the execution of this task